README.md

```diff
@@ -33,6 +33,13 @@ Data about food is of public interest and has to be open. The complete database
 
 * <https://world.openfoodfacts.org>
 
+## Roadmap
+* Extracting HTML templates from Perl code
+* Creating an API product route that can also answer for cosmetics using the Open Beauty Facts database
```

Comment:
That probably should be done somewhere else than in Product Opener. Either in the app, or through a proxy.

Reply:
can you propose a roadmap for outsiders ?

Reply:
Realistically, newcomers need to start with small things, small bugs etc. to get familiar with the code base. So things like the 5-minutes fix issues is probably a better way.